docs/getting_started.rst

```diff
@@ -54,7 +54,9 @@ Now you can then save the sky image to local disk e.g. FITS file format::
 
 or plot and analyse the sky image however you like.
 
-TODO: show how to plot the image with `astropy.visualization.wcsaxes`.
+The following plot was created using `astropy.visualization.wcsaxes`.
```

Comment:
Remove `.` at the end of the line and improve the text a bit, e.g. "If you execute the example above, you will get this sky image", i.e. make it clear how "this plot" relates to the example given in this secion.